fiji-openjdk-8/Dockerfile

```diff
@@ -8,16 +8,25 @@ FROM openjdk:8
 # Define maintainer.
 MAINTAINER Mark Hiner <hinerm@gmail.com>
 
+# Create a user.
+RUN useradd -u 1000 -ms /bin/bash fiji
+USER fiji
+
 # Define working directory.
-WORKDIR /fiji
+WORKDIR /home/fiji
```

Comment:
I would suggest to move that to /opt or any other dir. Home dir is likely to be overwiten by user.

Reply:
I don't have a strong opinion on where it lives. I put it under home because it kept the Dockerfile as short as possible. What type of usage do you foresee where `/home/fiji` is overwritten?

Reply:
on the shared environment like farm, singularity mount home dir https://singularity.lbl.gov/docs-mount#system-defined-bind-points

Reply:
Good to know, thanks. I'll update for a new location.